ddcz/templates/creative-pages/downloaditem-list.html

```diff
@@ -6,7 +6,7 @@
 
 {% block content %}
 <div id="page-common-article">
-    <h1 class="page-heading">{{ heading }}</h1>
+    <h1 class="page-heading text-red">{{ heading }}</h1>
```

Comment:
I wonder whether it doesn't make sense to add skin signifier for classes shared between skins, so `.historic-text-red`